editor/src/clj/editor/engine/native_extensions.clj

```diff
@@ -247,10 +247,13 @@
         ne-cache-info (query-cached-files server-url resource-nodes-by-upload-path evaluation-context)
         ne-cache-info-map (make-cached-info-map ne-cache-info)]
     (.accept builder #^"[Ljavax.ws.rs.core.MediaType;" (into-array MediaType []))
-    (if (not-empty user-info)
-        (.header builder "Authorization" (str "Basic " (.encodeToString (Base64/getEncoder) (.getBytes user-info)))))
-    (if (and (not-empty extender-username) (not-empty extender-password))
-        (.header builder "Authorization" (str "Basic " (.encodeToString (Base64/getEncoder) (.getBytes (str extender-username ":" extender-password))))))
+    (when (not-empty user-info)
+      (.header builder "Authorization" (str "Basic " (.encodeToString (Base64/getEncoder) (.getBytes user-info)))))
+    (when (and (not-empty extender-username) (not-empty extender-password))
+      (.header builder "Authorization" (str "Basic " (.encodeToString (Base64/getEncoder) (.getBytes (str extender-username ":" extender-password))))))
```

Comment:
Is it possible to have both `user-info` and `extender-*` authorization supplied here? Seems like it would result in two `Authorization` headers. Is that allowed?

Reply:
Yes, it is allowed, but I've read in a few places that the first header is the one that should be accepted and any subsequent headers with the same name should be ignored.